api/spec/requests/spree/api/states_spec.rb

```diff
@@ -4,7 +4,8 @@
 
 module Spree::Api
   describe 'States', type: :request do
-    let!(:state) { create(:state, name: "Victoria") }
+    let(:canada) { create(:country, iso: "CA") }
+    let!(:state) { create(:state, country: canada, name: "Victoria") }
```

Comment:
I'm pretty sure these states are from Australia, not Canada. 😉

Reply:
Changed the country to be Australia. Always good to have a local reviewer!